### PR TITLE
Merge pull request #4 from dharam4/Book_Navjeevan24

### PR DESCRIPTION
**Pull request by navjeevan24**
hi, @dipanshu42 I have updated the code in ReturnBook class, so can you please send request to my Reviewer @akay08923 for review. Thanks

### CODE REVIEW

**Comment by dipanshu42** [comment]
> hi, @dipanshu42 I have updated the code in ReturnBook class, so can you please send request to my Reviewer @akay08923 for review. Thanks

no worries i,ll let ankush know about this and the work will be delivered asap

**Comment by dipanshu42** [comment]
No worries i'll let ankush know about this and the review will be delivered
asap

File: library/returnBook/ReturnBookUI.java [hunk 1]
```diff
@@ -4,33 +4,51 @@
...
+			//switch (StATe) {
+			switch (state) {
 			
 			case INITIALISED:
 				break;
 				
 			case READY:
-				String BoOk_InPuT_StRiNg = iNpUt("Scan Book (<enter> completes): ");
+				//String BoOk_InPuT_StRiNg = iNpUt("Scan Book (<enter> completes): ");
+				String BookInputString = input("Scan book (<enter> completes): ");
```
Comment (akay08923) [inline library/returnBook/ReturnBookUI.java:51]: a variable name should be bookInputString instead of BookInputString 
@dipanshu42

**Review by akay08923 — CHANGES_REQUESTED** [review]
Hi @dipanshu42, i have reviewed the code which has been updated by @navjeevan24 . there was a one error on which i gave suggestion. Please let her know about this.

**Comment by dipanshu42** [comment]
Hey ankush
I have let navjeevan know about the change request and she will follow it
up asap

**Comment by navjeevan24** [comment]
hi dipanshu, I done changes in code so can you please informe to ankush for review.

**Comment by dipanshu42** [comment]
Ill let him know

**Comment by navjeevan24** [comment]
I changes code again dipanshu please request to ankush for review.

**Comment by navjeevan24** [comment]
I changes code again dipanshu please request to ankush for review.

**Comment by dipanshu42** [comment]
Already did that

**Comment by navjeevan24** [comment]
hi dharamveer, Dipanshu again update his code can you please review that and let me know if there any error or not. Thanks

Sent from my iPhone

Begin forwarded message:

**Review by akay08923 — APPROVED** [review]
hi @dipanshu42 , Everything is fine. @navjeevan24  can merge the code into master branch.

**Comment by navjeevan24** [comment]
ok I will merge it in master branch

**Comment by navjeevan24** [comment]
thanks

**Pull request merged**